src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const rent = 40;
```

Comment:
Consider using a constant with a descriptive name instead of the literal `40` to represent the daily rental cost. This will make the code more readable and maintainable.